config/locales/nl.yml

```diff
@@ -184,15 +184,15 @@ nl:
       dosage: dosis
       dosage_placeholder: 'b.v. 2'
       refill: bijvullen
-      comments_hint: 'Voeg hier additioneel commentaar toe, bijvoorbeeld je symptomen'
-      add_to_google_cal: 'Google Kalendar herinnering'
-      google_cal_hint: 'We voegen een herinnering toe voor dit medicijn in je Google kalendar.'
+      comments_hint: 'Voeg hier extra opmerkingen toe, bijvoorbeeld je symptomen'
+      add_to_google_cal: 'Google Agenda herinnering'
+      google_cal_hint: 'We voegen een herinnering toe voor dit medicijn in je Google Agenda.'
       strength_hint: 'Medicatie sterkte, meestal gevonden naast de naam'
       total_hint: 'Totale hoeveelheid pillen of hoeveelheid medicatie'
-      dosage_hint: 'Hoeveelheid pillen of dosis per dag'
-      weekly_dosage_hint: 'Dagen van de week om dit medicijn te nemen'
+      dosage_hint: 'Hoeveelheid pillen of dosissen per dag'
```

Comment:
Dosis of Dosissen dat is de vraag, om te vermijden kunnen we dosage gebruiken.

Reply:
het meervoud kan doses of dosissen zijn (http://www.meervoudvan.com/dosis/)